unittests/parser_test_case.py

```diff
@@ -84,3 +85,10 @@ def _test_calldef_exceptions(self, calldef, exceptions):
             (calldef.name,
              pprint.pformat([delc.name for delc in exception_decls]),
              pprint.pformat([delc.name for delc in exceptions_indeed])))
+
+if sys.version_info < (2, 7, 0):
```

Comment:
If you have unittest2 installed, I think this is not needed; there is a Travis build with python2.6 which is working, so I am wondering why this change is needed.